### PR TITLE
update CI to include Python 3.9rc1

### PR DESCRIPTION
**Pull request by nitzmahone**

### CODE REVIEW

**Comment by cclauss** [comment]
Travis CI build config validation issues...
Screenshot 2020-09-02 at 08 08 33

File: packaging/build/appveyor.ps1 [hunk 1]
```diff
@@ -87,10 +89,10 @@ Function Build-Wheel($python_path) {
     }
 
     # ensure pip is current (some appveyor pips are not)
-    Invoke-Exe { & $python -W "ignore:DEPRECATION" -m pip install --upgrade pip }
+    Invoke-Exe { & $python -W "ignore:DEPRECATION" -m pip install --no-warn-script-location --upgrade pip }
 
     # ensure required-for-build packages are present and up-to-date
-    Invoke-Exe { & $python -W "ignore:DEPRECATION" -m pip install --upgrade cython wheel setuptools --no-warn-script-location }
+    Invoke-Exe { & $python -W "ignore:DEPRECATION" -m pip install --no-warn-script-location --upgrade cython wheel setuptools --no-warn-script-location }
```
Comment (cclauss) [inline packaging/build/appveyor.ps1:95]: This is doing `--no-warn-script-location` twice.

Should we really be suppressing deprecation warnings?

**Comment by webknjaz** [comment]
@cclauss those are not issues, just informational messages. They aren't even warnings.

**Comment by cclauss** [comment]
I know what they are.  Providing the same command line option twice in one command is a source of confusion.

**Comment by webknjaz** [comment]
@cclauss I was responding to your Travis comment

File: .travis.yml [hunk 2]
```diff
@@ -20,17 +20,17 @@ matrix:
       env: TOXENV=py37
     - python: 3.8
       env: TOXENV=py38
-    - python: 3.8-dev
-      env: TOXENV=py38
+    - python: 3.9-dev
```
Comment (hugovk) [inline .travis.yml:23]: Python 3.9 final is now available:
```suggestion
    - python: 3.9
```

File: .travis.yml [hunk 3]
```diff
@@ -20,17 +20,17 @@ matrix:
...
+      env: TOXENV=py39
     - python: 3.7
       arch: arm64
       env: TOXENV=py37
     - python: 3.8
       arch: arm64
       env: TOXENV=py38
-    - python: 3.8-dev
+    - python: 3.9-dev
```
Comment (hugovk) [inline .travis.yml:31]:
```suggestion
    - python: 3.9
```

**Comment by ingydotnet** [comment]
Closing.

I believe this PR is covered by #407 

Reopen if I missed something.

**Pull request closed without merging**